datasette/utils/sqlite.py

```diff
@@ -0,0 +1,19 @@
+try:
+    import pysqlite3 as sqlite3
+except ImportError:
+    import sqlite3
+
+if hasattr(sqlite3, "enable_callback_tracebacks"):
+    sqlite3.enable_callback_tracebacks(True)
+
+
+def sqlite_version():
```

Comment:
I'm nervous about the performance impact of this, since every time it is called it creates a new in-memory connection.

I'm going to cache this so that it only ever gets called a maximum of once for the lifetime of the server.